examples/map/map_contouring.py

```diff
@@ -17,11 +18,17 @@
 aiamap = sunpy.map.Map(AIA_193_IMAGE)
 
 ###############################################################################
-# Now find the contours
-contours = aiamap.contour(50000)
+# In find a set of contours, we have to provide the level to contour in the
+# same units as the map data. To find out the units we can inspect
+# `sunpy.map.Map.unit`.
```

Comment:
This won't actually link to anything (as opposed to `sunpy.map.GenericMap.unit`)